.github/workflows/test-notify-build-status-integration.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: test-notify-build-status-integration
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    paths:
```

Comment:
Does it make sense to add the workflow itself?

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2256/commits/bd5129ee259a4d3d9bc95c14acc78775e2674b1a